tests/integration/src/disaster_recovery_tests.rs

```diff
@@ -418,6 +419,189 @@ fn test_disaster_recovery_flow() {
     assert!(dr_status.recovery_requests.is_empty());
 }
 
+// The goal of this test is not to check if the committee is set correctly, but to check that when the station is
+// reinstalled, the accounts are also recreated with the same addresses.
+#[test]
+fn test_disaster_recovery_flow_recreates_same_accounts() {
+    // 1. setup the environment with one admin user
+    let TestEnv {
+        env, canister_ids, ..
+    } = setup_new_env();
+
+    let station_wasm_module = get_canister_wasm("station");
+    let system_info = get_system_info(&env, WALLET_ADMIN_USER, canister_ids.station);
+    let upgrader_id = system_info.upgrader_id;
+    let admin_user = get_user(&env, WALLET_ADMIN_USER, canister_ids.station);
+
+    // 2. create 3 accounts with the same admin user and no approval required
+    let mut initial_accounts = BTreeMap::new();
+    for account_nr in 0..3 {
+        let create_account_args = AddAccountOperationInput {
+            name: format!("account-{}", account_nr),
+            blockchain: "icp".to_string(),
+            standard: "native".to_string(),
+            read_permission: AllowDTO {
+                auth_scope: station_api::AuthScopeDTO::Restricted,
+                user_groups: vec![],
+                users: vec![admin_user.id.clone()],
+            },
+            configs_permission: AllowDTO {
+                auth_scope: station_api::AuthScopeDTO::Restricted,
+                user_groups: vec![],
+                users: vec![admin_user.id.clone()],
+            },
+            transfer_permission: AllowDTO {
+                auth_scope: station_api::AuthScopeDTO::Restricted,
+                user_groups: vec![],
+                users: vec![admin_user.id.clone()],
+            },
+            transfer_request_policy: Some(RequestPolicyRuleDTO::AutoApproved),
+            configs_request_policy: Some(RequestPolicyRuleDTO::AutoApproved),
+            metadata: vec![],
```

Comment:
Maybe we could also make sure that the metadata are recovered properly (the ones set here discarded and replaced by those in the disaster recovery input).